src/AccessibilityInsights.Extensions/Interfaces/IssueReporting/IssueType.cs

```diff
@@ -1,11 +1,9 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
-namespace AccessibilityInsights.Extensions.Interfaces.BugReporting
+namespace AccessibilityInsights.Extensions.Interfaces.IssueReporting
 {
-    public enum  BugType
+    public enum  IssueType
     {
-        Unknown,
-        ColorContrast,
         SingleFailure,
```

Comment:
Please keep **Unknown** as the first item in the enum. The code should never set it, but it serves as a guard against places where we fail to set the enum

Reply:
it isn't used in master.

Reply:
That's correct. It exists to differentiate between intentionally set values and default (i.e., not set by the code owner) values. Common pattern